service/worker/scanner/executions/activities.go

```diff
@@ -1,459 +0,0 @@
-//
-// Permission is hereby granted, free of charge, to any person obtaining a copy
-// of this software and associated documentation files (the "Software"), to deal
-// in the Software without restriction, including without limitation the rights
-// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-// copies of the Software, and to permit persons to whom the Software is
-// furnished to do so, subject to the following conditions:
-//
-// The above copyright notice and this permission notice shall be included in all
-// copies or substantial portions of the Software.
-//
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-// SOFTWARE.
-
-package executions
-
-import (
-	"context"
-	"encoding/json"
-	"errors"
-	"time"
-
-	"go.uber.org/cadence"
-
-	"go.uber.org/cadence/.gen/go/shared"
-	"go.uber.org/cadence/activity"
-
-	c "github.com/uber/cadence/common"
-	"github.com/uber/cadence/common/metrics"
-	"github.com/uber/cadence/common/persistence"
-	"github.com/uber/cadence/common/reconciliation/invariant"
-	"github.com/uber/cadence/common/reconciliation/store"
-	"github.com/uber/cadence/service/worker/scanner/executions/shard"
-)
-
-const (
-	// ScannerConfigActivityName is the activity name ScannerConfigActivity
-	ScannerConfigActivityName = "cadence-sys-executions-scanner-config-activity"
-	// ScannerScanShardActivityName is the activity name for ScanShardActivity
-	ScannerScanShardActivityName = "cadence-sys-executions-scanner-scan-shard-activity"
-	// ScannerEmitMetricsActivityName is the activity name for ScannerEmitMetricsActivity
-	ScannerEmitMetricsActivityName = "cadence-sys-executions-scanner-emit-metrics-activity"
-	// FixerCorruptedKeysActivityName is the activity name for FixerCorruptedKeysActivity
-	FixerCorruptedKeysActivityName = "cadence-sys-executions-fixer-corrupted-keys-activity"
-	// FixerFixShardActivityName is the activity name for FixShardActivity
-	FixerFixShardActivityName = "cadence-sys-executions-fixer-fix-shard-activity"
-
-	// ErrScanWorkflowNotClosed indicates fix was attempted on scan workflow which was not finished
-	ErrScanWorkflowNotClosed = "scan workflow is not closed, only can run fix on output of finished scan workflow"
-	// ErrSerialization indicates a serialization or deserialization error occurred
-	ErrSerialization = "encountered serialization error"
-)
-
-var scanTypePrefixMap = map[ScanType]string{
-	ConcreteExecutionType: "", // leave it empty for now to be backwards compatible
-	CurrentExecutionType:  "current_executions_",
-}
```

Comment:
once landed please check the dashboard and update respectively

Reply:
sure